library/include/ck/library/tensor_operation_instance/gpu/grouped_gemm.hpp

```diff
@@ -256,6 +259,8 @@ struct DeviceOperationInstanceFactory<ck::tensor_operation::device::DeviceGroupe
                 add_device_grouped_gemm_xdl_splitk_f8_f16_f16_mk_kn_mn_irregular_instances(op_ptrs);
             }
         }
+#endif
+#if defined(CK_ENABLE_BF16) && defined(CK_ENABLE_INT8)
         else if constexpr(is_same_v<ADataType, bhalf_t> && is_same_v<BDataType, int8_t> &&
```

Comment:
If you add ifdefs at this level then just use `if ( ... )` - otherwise you could get an error if only this part of code would be compiled.
```suggestion
        if constexpr(is_same_v<ADataType, bhalf_t> && is_same_v<BDataType, int8_t> &&
```